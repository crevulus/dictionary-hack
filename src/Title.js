import React from 'react';

function Title () {
  return (
    <div className="title">
      <h1>ESL Dictionary</h1>
    </div>
  );
}

export default Title;
