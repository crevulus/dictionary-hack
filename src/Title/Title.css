.title {
  margin: auto;
  width: 90%;
  border-radius: 5px;
  background-color: #845EC2;
  text-align: center;
}