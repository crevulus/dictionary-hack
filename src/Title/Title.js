import React from 'react';
import './Title.css';

function Title() {
  return (
    <div className="title">
      <h1>ESL Dictionary</h1>
    </div>
  );
}

export default Title;
