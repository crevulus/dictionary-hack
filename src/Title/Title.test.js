
/* eslint-disable */
import React from 'react';
import ReactDOM from 'react-dom';
import { render } from '@testing-library/react';
import Title from './Title';
import renderer from 'react-test-renderer';

test('renders component', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Title></Title>, div);
});

test('matches snapshot', () => {
  const snap = renderer.create(<Title></Title>).toJSON();
  expect(snap).toMatchSnapshot();
});