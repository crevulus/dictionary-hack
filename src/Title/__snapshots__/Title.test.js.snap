// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="title"
>
  <h1>
    ESL Dictionary
  </h1>
</div>
`;
